RELEASE_NOTES.md

```diff
@@ -52,6 +52,10 @@ Please see the following for more details : https://docs.chef.io/release/12-6/re
 
 This is the first release where we are rolling out a MSI package for Windows that significantly improves the installation time. In a nutshell, the new approach is to deploy and extract a zipped package rather than individually tracking every file as a MSI component. Please note that the first  upgrade (ie, an older version of Chef client is already on the machine) may not exhibit the full extent of the speed-up (as MSI is still tracking the older files). New installs, as well as future upgrades, will be sped up. Uninstalls will remove the folder that Chef client is installed to (typically, C:\Opscode\Chef).
 
+## `windows_package` now supports non-`MSI` based installers
```

Comment:
Could you make a small change and add a " (Windows)" to the end of the title to be consistent with the other notes.

And sorry for the re-base .. hope it's not too painful.